src/odoc/indexing.ml

```diff
@@ -55,136 +52,103 @@ let compile_to_json ~output ~occurrences files =
     false
   in
   Format.fprintf output "[";
-  let _ : bool =
+  let _first =
```

Comment:
What's the reasoning behind only generating json fragments (and therefore having to pass around this 'first' thing) rather than generating a complete json object and simply outputting that?

Reply:
I think the idea when we did that was not to have the full object in memory. Now, I think that we anyway have it in memory. I'll remove the `first` thing.

Reply:
Done in [1e59b62](https://github.com/ocaml/odoc/pull/1251/commits/1e59b6201b1505008918983f4bfaf3b0bfb6897c)!